### ISSUE TITLE
Function failure for two monitors，one is full screen mode and the other is not full screen mode。

### ISSUE THREAD

**Issue by jinle94** [fikovnik/ShiftIt#249]
Function failure for two monitors， monitor A is full screen mode and monitor B is not full screen mode；on the monitor B，shiftlt is not work。

**Comment by fikovnik** [comment]
Did you try the latest beta from releases?

**Comment by bastula** [comment]
I tried the latest beta and I think it's an issue specific to Mac OS 10.12.6.

**Comment by fikovnik** [comment]
Interesting. I'm waiting for a new laptop so I cannot check it right now (currently on pc w/ linux).

**Comment by zoryamba** [comment]
same behavior on 10.13 high sierra

**Comment by jinle94** [comment]
only Iterm in full screen mode，issue appear，other app will be not，such as Safari。

**Comment by patrickoliveras** [comment]
Interestingly, all Shiftit commands work if the window being manipulated is the same app as the one in Full Screen mode. 

Example steps to replicate: 

1. Maximize a Chrome window.
2. Create a new Chrome window but in a Desktop.
3. Use Shiftit commands willi-nilly.

**Comment by namiwang** [comment]
confirmed on 10.13.1

**Comment by deldrid1** [comment]
+1 - I'm having the same issue on 10.13.2 and ShiftIt 1.6.4.  

I was on the 1.6.4 "beta" linked in #216 by @fikovnik  on Oct 21, 2016 and that had been working for several months.  I got pushed an update recommended by the ShiftIt app and then everything broke per this thread.  

Even though I am attempting to use ShiftIt on the non-full screen windows side (Fusion is running full screen on a second monitor), I get this error in the debug log

```
2017-12-08 08:32:25.434 ShiftIt[7399/0x7fff9d33c340] [lvl=3] -[ShiftItAppDelegate invokeShiftItActionByIdentifier_:] Execution of ShiftIt action: bottom failed: Windows in fullscreen are not supported
NSError stack trace:
org.shiftitapp.shifit.error:20103 - Windows in fullscreen are not supported
```

**Comment by jeep87c** [comment]
I'm having the same issue on 10.13.1 and ShiftIt 1.6.4.

Any update on this issue?

As for @deldrid1 it's working perfectly with the 1.6.4 beta version on an older OSX version (10.12.6).
I would have tried again the beta version but can't find it anymore...

**Comment by davidsielert** [comment]
Yeah I seem to have this issue whenever I have ANY app in full screen mode shiftit will not work at all. like to for instance keep a database window open in full screen in a space on a 3rrd monitor and when I do so it disables me from using shiftit in any other windows (chrome in my case)

**Comment by gwynnebaer** [comment]
@davidsielert - I can also confirm this:
Multiple monitors with one app full-screened - ShiftIt does not work at all.
Multiple monitors with no apps full-screened - ShiftIt works as designed.

This was new to me as I had not been using multiple monitors until recently.
